lib/screens/myAccount/ProfileScreen.dart

```diff
@@ -515,6 +515,12 @@ class _ProfileScreenState extends State<ProfileScreen> {
               : Colors.red);
           break;
 
+        case "OTG":
```

Comment:
text casenya di simpen ke dictionary aja gans

Reply:
oiyaaa siaap

Reply:
eh gabisa kang "Case expressions must be constant" katanya

Reply:
eh iya ya wkwk kirain bisa 😅 @jaluowen , di close dulu